class UserNotFoundException(Exception):
    pass


class BookNotFoundException(Exception):
    pass
